tests/resources/test_project.py

```diff
@@ -614,7 +614,7 @@ def test_list_projects_filters_non_projects(collection, session):
     session.set_response({'projects': projects_data})
 
     # Then
-    with pytest.raises(RuntimeError):
+    with pytest.raises(ValueError):
```

Comment:
And this change is based on the type of error thrown by `Property.deserialize_from_dict`.